src/main/features/core/tray.js

```diff
@@ -180,5 +180,50 @@ Emitter.on('audiooutput:list', (event, devices) => {
   setContextMenu();
 });
 
+// Change the Icon if its playing
+Emitter.on('playback:isPlaying', () => {
```

Comment:
IPC events could change across versions.  To be consistent we wrap these events in the internal `PlaybackAPI`

Can we change this to

```js
PlaybackAPI.on('change:state', () => {
```